test_runner/fixtures/neon_cli.py

```diff
@@ -496,6 +496,7 @@ def endpoint_create(
         internal_http_port: int,
         tenant_id: TenantId,
         pg_version: PgVersion,
+        grpc: bool = False,
```

Comment:
I'm thinking of how this will look like in the brave new world when we have switched completely to grpc. You don't want all tests to be passing an explicit `grpc=True` in that world. Perhaps this should default to None, meaning "use the default for the test suite", and only pass an explicit True/False in tests that are written specifically to test the grpc or libpq protocol.

(the above applies not only to this function, but everywhere where we now have a 'grpc=on/off' option)

Reply:
I figured we'd just remove the parameter in that world. But yeah, might be worth plumbing through `None` for the default anyway, will fix.